src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -538,6 +538,16 @@ def insertContainerRules(self):
                     rseName = rseName.replace("cms_type=real", "cms_type=test")
             elif self.testRSEs:
                 rseName = "%s_Test" % rseName
+            
+            #Checking whether we need to ask for rule approval
+            try:
```

Comment:
@amaltaro according to discussion over `requiresApproval` method, I'm adding this try clause here. I guess I could instead call the method inside the previously existing try clause (ln 554) but I like it better this way. What do you think?

Reply:
This try/except here looks good. Most of the error message is already encoded in the exception though, so the error message here can likely be shorter and append str(exc) etc.

Reply:
I shortened the message